.github/workflows/lint.yaml

```diff
@@ -4,10 +4,9 @@ on:
   push:
     branches:
       - "!dependabot/*"
-      - "*"
+      - "main"
```

Comment:
Why was this changed?

Reply:
So we don't get duplicated CI builds running in a PR. It's the same pattern we use in other repos.

Reply:
Can you put that as a distinct commit then and explain that in the message?

Reply:
addressed in https://github.com/authzed/api/pull/40/commits/1a3254e5062a5d65b09a8ef048ca38d6f2464adb

Reply:
Ideally, development would be done on a fork and this value should be `*` so that if we were to produce a v2 branch etc... it would run CI correctly.